presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoPreparedStatement.java

```diff
@@ -538,6 +538,9 @@ else if (x instanceof byte[]) {
         else if (x instanceof Date) {
             setDate(parameterIndex, (Date) x);
         }
+        else if (x instanceof LocalDate) {
+            setAsDate(parameterIndex, x);
```

Comment:
maybe function should be called `setAsLocalDate` or `setLocalDate` to match the parameter type?

Reply:
`setAsDate` takes Object and actually works with several different date representations